fbuapp/View Controllers/FilterViewController.m

```diff
@@ -155,7 +164,19 @@ -(BOOL)isValidNumber: (NSString *) string{
 }
 
 - (void)didSelectCell: (NSIndexPath *)indexPath{
-    self.eventCategory = indexPath.row;
+    [self.eventCategory addObject:@(indexPath.row)];
+}
+
+- (void)didDeselectCell: (NSIndexPath *)indexPath{
+    [self.eventCategory removeObject:@(indexPath.row)];
+    if([self.eventCategory count] > 0)
+    {
+        [self.postQuery whereKey: @"eventCategory" containedIn:self.eventCategory];
+        
+    }
+    else{
+        [self.postQuery whereKey: @"eventCategory" containedIn:@[@0, @1, @2, @3, @4, @5, @6, @7]];
```

Comment:
is there a way to clear the query for "eventCategory" without having to list out the entire array 0 to 7

Reply:
In order to override the previous whereKey containedIn query on postQuery we must use the same query method, so this is the only way